src/pages/SetupPlot.vue

```diff
@@ -192,6 +198,10 @@ export default defineComponent({
       const result = await native.selectDir(this.plotDirectory).catch(console.error)
       if (result) this.plotDirectory = result
     },
+    async initFarming(path: string) {
+      const public_key = await startFarming(path)
+      return public_key
+    },
```

Comment:
```suggestion
    initFarming(path: string) {
      return startFarming(path)
    },
```

No idea about Vue, but I'd really like to see return types in all these functions as well.